apps/deploy-web/tests/ui/fixture/Leap/async/2871.js

```
@@ -444,14 +454,14 @@ _global.SENTRY_RELEASE = { id: "0.22.9" };
         m = Object.prototype.hasOwnProperty,
         u = Object.prototype.propertyIsEnumerable,
         x = (e, a, t) => (a in e ? i(e, a, { enumerable: !0, configurable: !0, writable: !0, value: t }) : (e[a] = t)),
-        h = (e, a) => {
+        p = (e, a) => {
           for (var t in a || (a = {})) m.call(a, t) && x(e, t, a[t]);
           if (d) for (var t of d(a)) u.call(a, t) && x(e, t, a[t]);
```

Biome (2.1.2)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)